docs/utime.rst

```diff
@@ -0,0 +1,159 @@
+utime
+*****
+
+.. py:module:: utime
+
+The ``utime`` module provides functions for getting the current time and date, 
+measuring time intervals, and for delays.
+
+.. note::
+    The ``utime`` module is a MicroPython implementation of the standard Python 
+    ``time`` module. It can be imported using both ``import utime`` and 
+    ``import time``, but the module is the same.
+
+
+Functions
+=========
+
+.. method:: utime.sleep(seconds)
+
+    Sleep for the given number of seconds. Some boards may accept seconds as a 
+    floating-point number to sleep for a fractional number of seconds. Note that 
+    other boards may not accept a floating-point argument, for compatibility 
+    with them use :func:`utime.sleep_ms()` and :func:`utime.sleep_us()`
+    functions.
```

Comment:
Can we check if the micro:bit port accepts floating point numbers and update this paragraph accordingly?